packages/migrate/src/utils/setupPostgres.ts

```diff
@@ -53,3 +53,21 @@ export async function tearDownPostgres(options: SetupParams) {
   `)
   await db.end()
 }
+
+/**
+ * Run an arbitrary query against a Postgres database.
+ * Procedural `plpgsql` queries are also supported.
+ * This function should be called after `setupPostgres` and before `tearDownPostgres`.
+ * The given `options.connectionString` is used as is.
+ */
+export async function runQueryPostgres(options: SetupParams, query: string): Promise<void> {
+  const { connectionString } = options
+
+  // Connect to default db
+  const dbDefault = new Client({
+    connectionString,
+  })
+  await dbDefault.connect()
+  await dbDefault.query(query)
+  await dbDefault.end()
+}
```

Comment:
In the future, we may want to have similar functions such as this, but for other providers.